packages/apps/stock/src/screens/internalMoves/InternalMoveLineCreationScreen.js

```diff
@@ -68,14 +68,20 @@ const InternalMoveLineCreationScreen = ({navigation}) => {
 
   const handleToProductTrackingNumberChange = useCallback(
     _value => {
+      if (destinationStockLocation != null) {
+        setStockLocationLine(_value);
+        setMovedQty(0);
+        return;
+      }
+
```

Comment:
This has still no functional meaning ! **Do not skip checks !**
The changes needed here have to be done in helper functions (`handleReset` & `handleNextStep`).
If checks are there, it is for a reason, you cannot skip them whenever you want or you will lose functionality (`currentStep` value change for exemple here...). Moreover `handleReset` & `handleNextStep` enable to standardize the code and make it more readable : 
* If value is null then we are currently doing a reset of the step --> functional behaviour of reset of this step is done in `handleReset`, not at two places at the same time...
* If value is not null, then we need to go to next step --> functionnal behaviour of going to next step is done in `handleNextStep`, nowhere else